sys/filesys_windows.go

```diff
@@ -257,12 +260,70 @@ func windowsOpenSequential(path string, mode int, _ uint32) (fd windows.Handle,
 	return h, e
 }
 
-// ForceRemoveAll is the same as os.RemoveAll, but uses hcsshim.DestroyLayer in order
-// to delete container layers.
+// ForceRemoveAll is the same as os.RemoveAll, but is aware of io.containerd.snapshotter.v1.windows
```

Comment:
I think clarifying in the comment that the special `io.containerd.snapshotter.v1.windows` behavior will only work if `ForceRemoveAll` is called on the root containerd data directory path.

Reply:
https://github.com/containerd/containerd/compare/7cfb685d725e3e963eb9403b978383b405594227..e5968e2dffce389a047c16d1eddfbe169962dbd0